scripts/nix/fetchNixpkgs.nix

```diff
@@ -0,0 +1,43 @@
+{ owner
```

Comment:
We don't need this file.

Reply:
Appears to be used by cardano-sl, not sure how much of their nix configs we should use.